frontend/test/__runner__/run_cypress_tests.js

```diff
@@ -68,6 +56,17 @@ const init = async () => {
   await generateSnapshots();
 
   printBold("Starting Cypress");
+  if (!isOpenMode) {
+    printYellow(
+      "If you are developing locally, prefer using `yarn test-cypress-open` instead.\n",
+    );
+  }
+
+  const logMessage = isFolderFlag
+    ? `Running tests in '${sourceFolderLocation}'`
+    : `Running '${testFiles}'`;
+
+  printBold(logMessage);
```

Comment:
This make the order of steps more clear.

It's roughly:
1. get version
2. start backend
3. genearte snapshosts
4. run Cypress tests
5. stop backend and cleanup